### ISSUE TITLE
warning message with stream: true

### ISSUE THREAD

**Issue by Paxa** [brianmario/mysql2#627]
I run:

```ruby
result = client.query(sql, stream: true, symbolize_keys: true)
# warning: cacheRows is ignored if streaming is true
```

To remove warning need to add: `cache_rows: false`

```ruby
result = client.query(sql, stream: true, symbolize_keys: true, cache_rows: false)
```

I think warning message should be:

```
warning: cache_rows is ignored if streaming is true
```

https://github.com/brianmario/mysql2/blob/435b550fe751322ace29a1155cd57c353c072055/ext/mysql2/result.c#L874

**Comment by sodabrew** [comment]
Thank you, this is a great suggestion.

**Comment by sodabrew** [comment]
Subtle note to self: If the user did not explicitly set cache_rows: true, don't warn here.

**Comment by sodabrew** [comment]
Resolved by #630
